### ISSUE TITLE
Deno is coming

### ISSUE THREAD

**Issue by DefinitelyMaybe** [retejs/rete#435]
Howdy!

throwing around ideas here, I was wondering your thoughts on use-of or moving-to [Deno](https://deno.land/).

They're also currently on track to releasing a 1.0. [heres](https://github.com/denoland/deno/issues/2473) a link to the issue

**Comment by Ni55aN** [comment]
https://en.reactjs.org/docs/refs-and-the-dom.html

```js
  componentDidMount() {
    createEditor(this.ref); // createEditor accepts a container for editor 
  }
```

**Comment by SirWindfield** [comment]
@DefinitelyMaybe Shouldn't it just be fire-and-forget in the best case? Seems like it's just another runtime (which does natively run Typescript). So as long as the application is written in TS and would compile without errors, deno should be able to run it.

Maybe I'm just missing something here though

**Comment by jeanlescure** [comment]
I'd suggest adding tests to make sure Deno can run your code. In my experience this makes for much more resilient code as Deno uses very strict and secure standards.

Just to give an example, up to recently one of my libs was importing a JSON file, this is not good practice and sure enough Deno did not like it when I imported the url to the TS file with the mentioned JSON import, and thus I refactored the lib to use a different approach.

Deno + TSLint are a match made in heaven for truly safe and well formatted TypeScript :ok_hand:
